### PR TITLE
Avoid a possible divide-by-zero error

### LINKED ISSUE #55: Division by zero at /vendor/szymach/c-pchart/src/Draw.php:2750
I have wiped all labels and axis information to create just pure graphic charts. [See example of graph style here](http://bourbon.hipstercreative.com/101H0B0D203f). Simple math error because I have 0 points to draw on.

### PR DESCRIPTION
**Pull request by lhilton**
Fixes #55

### CODE REVIEW

File: src/Draw.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
  *  Version     : 2.1.4
  *  Made by     : Jean-Damien POGOLOTTI
  *  Last Update : 19/01/2014
- *
+ *©
```
Comment (szymach) [inline src/Draw.php]: Please remove this character.

File: src/Draw.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 /**
  *  Draw - class extension with drawing methods
- *
+ *©
```
Comment (szymach) [inline src/Draw.php]: :D

Comment (lhilton) [reply src/Draw.php]: Yup, as soon as my push went up I saw it. Nothing is ever easy, doubly so when your own little scripts start giving you attitudes. I'm glad this ended up in comments. Now I need to scour the rest of my code from the last week to see where else this has been injected.

**Comment by szymach** [comment]
Thank you

**Pull request merged**